storage/src/tests/distributor/bucketdbupdatertest.cpp

```diff
@@ -1474,10 +1474,6 @@ TEST_F(BucketDBUpdaterTest, pending_cluster_state_send_messages) {
               getSentNodes("distributor:10 storage:2",
                            "distributor:10 .1.s:d storage:2"));
 
-    EXPECT_EQ(getNodeList({1}),
-              getSentNodes("distributor:2 storage:2",
-                           "distributor:2 storage:2 .1.d:3 .1.d.1.s:d"));
-
```

Comment:
Should the test assertion below it be removed as well? It also mentions an explicit disk state

Reply:
Removed.